src/components/layout/header/basicMenu/index.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { IconButton, Menu, MenuItem, Button } from '@mui/material';
+import LanguageIcon from '@mui/icons-material/Language';
+
+const BasicMenu = () => {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div>
+      <IconButton
+      id="basic-button"
+      aria-controls={open ? 'basic-menu' : undefined}
+      aria-haspopup="true"
+      aria-expanded={open ? 'true' : undefined}
+      onClick={handleClick}>
+        <LanguageIcon></LanguageIcon>
+      </IconButton>   
+      <Menu
+        id="basic-menu"
+        anchorEl={anchorEl}
+        open={open}
+        onClose={handleClose}
+        MenuListProps={{
+          'aria-labelledby': 'basic-button',
+        }}
+      >
+        <MenuItem onClick={handleClose}>한국어</MenuItem>
```

Comment:
```suggestion
        <MenuItem onClick={handleClose}>한국어</MenuItem>
        <MenuItem onClick={handleClose}>日本語</MenuItem>
        <MenuItem onClick={handleClose}>English</MenuItem>
```
현재 예정 상으로는 구현하지 않을 기능이라 없애거나 주석 처리해둬도 괜찮을 것 같습니다.